automation/check-patch.k8s-1.11.0-release.yumrepos

```diff
@@ -0,0 +1 @@
+check-patch.yumrepos
```

Comment:
should be k8s-1.13.3

Reply:
Yeah, when I originally created the file, hadn't updated yet.